packages/core/test/insured-bridge/L2_DepositBox.js

```diff
@@ -0,0 +1,52 @@
+const hre = require("hardhat");
```

Comment:
this initial test file shows how: a) we can continue to write tests in web3 b) we can _also_ use smockit tests to mock L1<->L2 communication.